docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+
+Consider the art of the possible and use what you can&mdash;Zarf will continue to work, regardless.
+
+&nbsp;
+
+
+### Appliance Mode
+
+<a href="../.images/what-is-zarf/appliance-mode.png">
+  <img alt="how it works" src="../.images/what-is-zarf/appliance-mode.png" height="295" />
+</a>
+
+#### (0) - Package
+
+Move a Zarf release + your desired packages to your Zarf cluster machine.
+
+#### (1) - Zarf cluster
+
+Make your machine into a **single node** Zarf cluster with the command: `zarf init`.
+
+Recommended Zarf components: `management`.
+
+Deploy your package into the Zarf cluster with the command: `zarf deploy`.
+
+#### (✓) - Use it
+
+Connect directly to the Zarf cluster to access your newly unpackaged applications!
+
+&nbsp;
+
+
+### Utility Mode
+
+<a href="../.images/what-is-zarf/utility-mode.png">
+  <img alt="how it works" src="../.images/what-is-zarf/utility-mode.png" height="283" />
+</a>
+
+#### (0) - Package
+
+Move a Zarf release + desired packages to your Zarf cluster machine.
+
+#### (1) - Zarf cluster
+
+Make your Zarf cluster machine into a **single node** Zarf cluster with the command: `zarf init`.
+
+Recommended Zarf components: `management` & `gitops-services`.
+
+Deploy your package into the Zarf cluster with the command: `zarf deploy`.
+
+#### (2) - Downstream cluster
+
+Point your downstream cluster to the Zarf cluster **source code** & **container image repository** for access to base software & updates (hint: [GitOps](https://www.weave.works/technologies/gitops/)).
+
+#### (✓) - Use it
+
+Once your downstream cluster has settled accounts with the Zarf cluster, connect to the downstream cluster to access your newly unpackaged applications!
+
+&nbsp;
+
+
+## Target Use Cases
+
+Zarf's possibilities are endless&mdash;Zarf developers' time is not. Thus, scope definition is in order.
+
+Here are the things we think Zarf should get really good at, listed top-down in order of importance:
+
+- Make movement of software "across the air gap" an open source "solved problem".
+
+- Make it trivial to deploy & run Kubernetes apps "at the Edge".
+
+- Make it easy to support GitOps-based k8s cluster updates in isolated environments.
+
+- Make it possible to support GitOps-based k8s cluster updates in internet-connected-but-independent environments (think: dependency caching per availability zone, etc).
+
+&nbsp;
+
+
+## What can be packaged?
+
+To reiterate: Zarf's possibilities are endless&mdash;Zarf developers' time is not. Thus, scope definition is again in order.
+
+Given Zarf's being a "k8s cluster to serve _other_ k8s clusters", the following types of software can be rolled into a Zarf package:
+
+- container images &mdash; to serve images for the Zarf & downstream clusters to run containers from.
+
+- git repositories &mdash; to serve as the git-based "source of truth" for downstream "GitOps"ed k8s clusters to watch.
+
+- pre-compiled binaries &mdash; to provide the software necessary to start & support the Zarf cluster.
+
```

Comment:
Not just pre-compiled binaries, that's a particular file type zarf is aware of, but could also be infra code, ie. terraform modules, scripts, config files, etc.  Also big one that was left off is the data injection one

Reply:
Mmm... lemme make sure I'm clear on this one:

You're suggesting we add another "type of software" line in here to cover stuff that, 1) gets installed on the Zarf cluster machine, but 2) is _not_ a binary?  Stuff for Zarf to use to bootstrap other stuff?

Reply:
Also, where would the "data injection" part fit?  I sort of imagined that was covered by the _container images_ line item but you're thinking it should be it's own thing?

Reply:
Oh, unless you're thinking that we should call out that you can use Zarf to package stuff that has _nothing_ to do with the Zarf cluster at all.  So, like, to add an additional block that talks about if you just want to use the packaging facilities to seed your own non-Zarf-cluster stuff (i.e. TF modules for your own infra, etc.).

Reply:
Bingo